tests/integration/apiCollection.test.js

```diff
@@ -23,6 +23,9 @@ async function main() {
         "PerformanceTiming.prototype.navigationStart",
```

Comment:
What this test does is it goes to https://privacy-test-pages.glitch.me/privacy-protections/fingerprinting/ and then checks if crawler observed all API calls. Some calls however are expected not to be observed e.g. because:
- we don't call them on that fingerprinting page (e.g. cookie access)
- they require user action (touch events)
- they require sensors to be available in the browser (e.g gyroscope)
- they are not collected due to a bug (e.g. matchMedia) 

This time I added just couple of entries - cookieStore (because fp page is not testing for storage options) and matchMedia (there is some kind of a bug that doesn't allow us to test various matchMedia options - we can only test one)